resync/workflows/workflow_capacity_forecasting.py

```
@@ -636,12 +638,22 @@ async def run_capacity_forecast(
-    llm = LLMFactory.get_langchain_llm(model="claude-3-sonnet-20240229") # Updated to valid model ID
+    from resync.settings import settings
+    
+    model_name = getattr(settings, "agent_model_name", None) or getattr(settings, "llm_model", "gpt-4o")
+    llm = LLMFactory.get_langchain_llm(model=model_name)
 
     async with get_async_session() as db:
-        checkpointer = PostgresSaver(db.connection())
+        if POSTGRES_SAVER_AVAILABLE and PostgresSaver is not None:
+            try:
+                checkpointer = PostgresSaver(db.connection())
+            except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)